.github/workflows/Windows.yml

```diff
@@ -0,0 +1,29 @@
+name: Windows Tests
+env:
+  COVERALLS_REPO_TOKEN: ${{ secrets.COVERALLS_REPO_TOKEN }}
+
+on:
+  pull_request:
+
+jobs:
+  test:
+    runs-on: windows-latest
+    steps:
+      - name: Configure git
+        run: "git config --global core.autocrlf false"
+        shell: bash
+      - name: Checkout
+        uses: actions/checkout@v2
+      - name: Setup Scala
+        uses: olafurpg/setup-scala@v10
+      - name: Cache
+        uses: actions/cache@v2
+        with:
+          path: |
+            ~/.ivy2/cache
+            ~/.sbt
+            ~/.bitcoin-s/binaries
+          key: ${{ runner.os }}-cache
+      - name: Windows Crypto, Core, and Database tests
+        run: sbt ++2.13.4 cryptoTest/test coreTest/test dbCommonsTest/test
```

Comment:
it probably wouldn't be a bad idea to run the `bitcoindRpcTest` and `eclairRpcTest` stuff too? There some file manipulation in those projects too.

Reply:
Added in a separate workflow. Originally put it in same but the single CI run took over 30 mins

Reply:
Looks like even on it's own it's extremely slow